### ISSUE TITLE
Supporting virtualization of Large options list?

### ISSUE THREAD

**Issue by cheahkhing** [furqanZafar/react-selectize#43]
When I have an options list of over 10000 items for a SimpleSelect or MultiSelect, it will be slow when rendering them on first load. Is there a plan to support virtualization of the list? so that only those visible options are rendered on the fly, it will improve the performance as well.

**Comment by furqanZafar** [comment]
maybe in release 1.0, using this: https://bvaughn.github.io/react-virtualized/

**Comment by cheahkhing** [comment]
This will be great. :) totally looking forward to this.

**Comment by cheahkhing** [comment]
Hi, i would like to get started on this item. I am trying to see how i can integrate them into react-selectize. But seems like the changes might be quite big. I am actually doing it in normal js file instead of your livescript (since i am not familiar with this). Just wondering if you have plan on this?

**Comment by furqanZafar** [comment]
Nope, I didn't get time to work on this, I am writing a Babel / ES6 version of react-selectize but not getting much free time to complete it still

**Comment by bvaughn** [comment]
Hello folks. 😄 

If there's any interest in [react-virtualized](https://bvaughn.github.io/react-virtualized/) integration I'd be happy to help. Were you thinking of built-in support or more of a HOC approach like [react-virtualized-select](https://github.com/bvaughn/react-virtualized-select)?

**Comment by julien-f** [comment]
@furqanZafar Any news on this?
